plugins/systemadmin/systemadmin.plugin.zsh

```diff
@@ -140,12 +140,16 @@ d0() {
 
 # gather external ip address
 geteip() {
-    curl http://ifconfig.me
+    echo "$(curl -s -S http://ipecho.net/plain)"
 }
 
 # determine local IP address
 getip() {
-    ifconfig  | grep 'inet addr:'| grep -v '127.0.0.1' | cut -d: -f2 | awk '{ print $1}'
+    if [ $(hash | grep '^ip=') ]; then
```

Comment:
What does hash output? It is a zsh builtin which shouldn't print anything by itself...